tracer/test/Datadog.Trace.Tests/DataStreamsMonitoring/DataStreamsContextPropagatorTests.cs

```diff
@@ -42,24 +42,16 @@ public void CanRoundTripPathwayContext()
         [Fact]
         public void Inject_WhenLegacyHeadersDisabled_DoesNotIncludeBinaryHeader()
         {
-            Environment.SetEnvironmentVariable("DD_DATA_STREAMS_LEGACY_HEADERS", "false");
-            try
-            {
```

Comment:
The changes in this file are just removing the `Environment` calls and `try-finally`, and passing the boolean in directly